spec/views/post/index.html.erb_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts index view', type: :system do
+  describe 'Index page' do
+    subject { User.create(name: 'Ashley', photo: 'https://www.anisearch.de/images/character/cover/full/0/820.webp', bio: 'Teacher from mexico.', posts_counter: 0) }
+
+    it 'Display posts view content' do
+      post = Post.create(author_id: subject.id, title: 'Hello 1', text: 'This is my post', comments_counter: 0,
+                         likes_counter: 0)
+      Post.create(author_id: subject.id, title: 'Hello 2', text: 'This is my post 2', comments_counter: 0,
+                  likes_counter: 0)
+      Post.create(author_id: subject.id, title: 'Hello 3', text: 'This is my post 3', comments_counter: 0,
+                  likes_counter: 0)
+      Comment.create(author_id: subject.id, post_id: post.id, text: 'Test comment')
+      visit user_posts_path(subject)
+
+      expect(page).to have_selector('img')
+      expect(page).to have_content('Ashley')
+      expect(page).to have_content('Number of posts')
+      expect(page).to have_content('Hello 1')
+      expect(page).to have_content('This is my post')
+      expect(page).to have_content('Test comment')
+      expect(page).to have_content('Comments: 1')
+      expect(page).to have_content('Likes: 0')
+
+      click_link('Hello 1')
+      sleep(5)
+      expect(page).to have_current_path(user_post_path(subject, post))
```

Comment:
- [ ] Let's write more descriptive tests. The listed assertions in your project requirement should be wrapped within an "it" block. For example:

  ```
  describe "Index page" do
    ...

    context "When a user visits the post index page" do
      ...
      it "you can see the user's profile picture" do
        expect(...) to...      # assertion that demonstrates user can see profile picture
      end

      it "you can see the number of posts the user has written" do
        expect(...) to...     # assertion that demonstrates the user can see the number of posts
      end
      ...
    end
  end
  ```